ksp_plugin_adapter/reference_frame_selector.cs

```diff
@@ -140,27 +145,33 @@ public enum FrameType {
                                  CelestialBody selected,
                                  Vessel target_override) {
     if (target_override) {
-      return "Tgt LVLH@" + selected.name[0];
+      return Localizer.Format("#Principia_RFS_NavBallText_Target",
```

Comment:
Principia_ReferenceFrameSelector_ShortName_Target, and same below in this function,
Principia_ReferenceFrameSelector_ShortName_BodyCentredNonRotating etc.